### PR TITLE
dlgInsertcolumns error fix

### PR DESCRIPTION
**Pull request by shadrackkibet**

### CODE REVIEW

File: instat/dlgInsertColumn.vb [hunk 1]
```diff
@@ -54,6 +54,11 @@ Public Class dlgInsertColumn
         ucrPnlBeforeAfter.SetParameter(New RParameter("before", 1))
         ucrPnlBeforeAfter.AddRadioButton(rdoBefore, "TRUE")
         ucrPnlBeforeAfter.AddRadioButton(rdoAfter, "FALSE")
+        ucrPnlBeforeAfter.AddParameterValuesCondition(rdoBefore, "before", "TRUE")
+        ucrPnlBeforeAfter.AddParameterValuesCondition(rdoBefore, "before", "FALSE", False)
+        ucrPnlBeforeAfter.AddParameterValuesCondition(rdoAfter, "before", "FALSE")
+        ucrPnlBeforeAfter.AddParameterValuesCondition(rdoAfter, "before", "TRUE", False)
```
Comment (dannyparsons) [inline instat/dlgInsertColumn.vb]: The second conditions you have for each radio button are True by definition if the first condition is True so they are not needed.

File: instat/dlgInsertColumn.vb [hunk 2]
```diff
@@ -81,13 +83,7 @@ Public Class dlgInsertColumn
...
         ucrPnlInsertColumns.AddParameterValuesCondition(rdoAtStart, "before", "TRUE")
-        ucrPnlInsertColumns.AddParameterValuesCondition(rdoAtStart, "before", "FALSE", False)
-        ucrPnlInsertColumns.AddParameterPresentCondition(rdoAtStart, "adjacent_column", False)
-
         ucrPnlInsertColumns.AddParameterValuesCondition(rdoAtEnd, "before", "FALSE")
-        ucrPnlInsertColumns.AddParameterValuesCondition(rdoAtEnd, "before", "TRUE", False)
-        ucrPnlInsertColumns.AddParameterPresentCondition(rdoAtEnd, "adjacent_column", False)
-
         ucrPnlInsertColumns.AddParameterValuesCondition(rdoBeforeAfter, "before", "TRUE")
         ucrPnlInsertColumns.AddParameterValuesCondition(rdoBeforeAfter, "before", "FALSE")
```
Comment (dannyparsons) [inline instat/dlgInsertColumn.vb]: These two are not correct - we can't have conditions checking if the value is two things at the same time

**Comment by stevenndungu** [comment]
@dannyparsons this is ready for you.
Note: insert column panel resets on reopen. We have set the conditions to what we deem is right but still resets.. Perhaps you could help on that.
-

**Comment by shadrackkibet** [comment]
@dannyparsons  this is ready for you.

**Comment by dannyparsons** [comment]
Thanks, it's on my list. Keep reminding me if I don't get to it.

**Comment by shadrackkibet** [comment]
@dannyparsons  this is urgent for the release. Our previous release had a developer error and  controls disappearing on reopen which i fixed on this pullrequest.

**Pull request merged**